CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ In order to get started contributing code, make sure you have the following inst
 
 This project is made up of a [command line tool](https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/tree/master/src/DacpacTool) that does most of the heavy lifting and an [accompanying NuGet package](https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/tree/master/src/MSBuild.Sdk.SqlProj) that puts it all together. When you make changes to the `DacpacTool` you'll want to run a `dotnet build` in the `src/MSBuild.Sdk.SqlProj` folder. This ensures you're using the latest bits when building any of the [test projects](https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/tree/master/test). There is a set of [unit tests](https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/tree/master/test/DacpacTool.Tests) for the command line tool as well. For new features it is recommended to add new test projects in the `test` folder to ensure that the functionality works as intended.
 
-If you want to debug the project locally we recommend making a change to the Main method in [Program.cs](https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/blob/master/src/DacpacTool/Program.cs) by adding `System.Diagnostics.Debugger.Launch()` as the first statement. This should provide you with the opportunity to attach a debugger to the tool so that you can step through.
+If you want to debug the `DacpacTool` locally you can run `dotnet build /p:MSBuildSdkSqlProjDebug=True` on a project that uses the SDK. This will trigger the `DacpacTool` to wait for a debugger to attach before doing any actual work. Note that this only works when the `DacpacTool` is compiled for the `Debug` configuration.
```

Comment:
Just one quick question on this - when you say "this only works when the `DacpacTool` is compiled for the `Debug` configuration", exactly what does that involve? Here's how I interpret it, but let me know if I'm understanding correctly:

1. The user needs to build `DacpacTool` (and potentially the whole Sdk) locally in the `Debug` configuration
2. The Sdk compiled for `Debug` needs to be installed in the local Nuget repository
3. The project to be debugged needs its Sdk reference updated for the locally-build version

Am I over-complicating things, and maybe it's simpler than that? Either way, I feel like it would help to spell out more clearly exactly what the process is for getting this to work.

Reply:
Yeah, good point. I will try to make this a little bit clearer.